packages/lesswrong/server/scripts/triggerWrappedRefresh.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-console */
+/* eslint-disable no-useless-escape */
+import { getAnalyticsConnectionOrThrow } from "../analytics/postgresConnection";
+import { Globals } from "../vulcan-lib";
+
+const USER_ENGAGEMENT_VIEW_NAME = 'user_engagement_wrapped_2023';
+const USER_ENGAGEMENT_VIEWDEF = `
+WITH normalized AS (
+  SELECT
+    raw.environment,
+    raw."timestamp",
+    raw.event ->> 'userId'::text AS user_id,
+    raw.event ->> 'path'::text AS path,
+    CASE WHEN (raw.event ->> 'increment'::text) ~ '^[0-9\.]+$'::text THEN
+      (raw.event ->> 'increment'::text)::numeric
+    ELSE
+      0::numeric
+    END AS increment
+  FROM
+    raw
+  WHERE
+    raw.event_type = 'timerEvent'
+    AND raw.environment = 'production'
+    -- DEBUG: Uncomment below to run over a shorter time period for faster
+    -- AND raw."timestamp" >= '2023-12-01 00:00:00'::timestamp WITHOUT time zone
+    AND raw."timestamp" < '2023-01-01 00:00:00'::timestamp WITHOUT time zone
```

Comment:
In testing I haven't actually run it over the full time range yet because it takes so long, I'm letting it run overnight now